understanding/about.html

```diff
@@ -72,22 +71,7 @@ <h3>Structure of the Documents</h3>
     </section>
 				
 			</section>
-			<section id="changelog">
-                <details>
-                    <summary>
-                        <h2>Change Log</h2>
-                    </summary>
-                    <p>A list of significantly updated Understanding documents since WCAG 2.1 was published:</p>
-                    <ol>
-                      <li>
-                        <time datetime="2019-01-17">Jan 18th 2019</time>: Updated <a href="./non-text-contrast">Understanding Non-text contrast</a>, based on the changes from <a href="https://github.com/w3c/wcag/pull/550">Pull request 550</a>.
-                      </li>
-                    </ol>
-                    <p>For a more detailed view of recent changes to the informative documents see the <a href="https://github.com/w3c/wcag/commits/master">github updates</a>.</p>
-          
-                </details>
-            </section>
-
+			
```

Comment:
Why is the changelog section deleted?
It is important to be able to retrieve when which piece was changed.

Reply:
Because this has not been maintained at all in the last 5 years. It only contains a single item from one small change in 2019. The actual change/commit log is still available on github. Not sure it's worth keeping the section around at all, even if we removed that solitary single list item and just kept the reference to github.

Reply:
I think it is valuable to keep the reference, so interested people can find the change log.

Reply:
so just changing it to

```html
<details>
    <summary>
        <h2>Change Log</h2>
    </summary>
    <p>For a detailed view of recent changes to the informative documents see the <a href="https://github.com/w3c/wcag/commits/master">github updates</a>.</p>
</details>
```

(and leaving the link further up in the page to the change log section back in) ?

Reply:
I've done the above (reintroduced the section, but only to point to github and nothing more)